README.md

```diff
@@ -7,21 +7,19 @@ See the [nf-encyclopedia documentation](https://TalusBio.github.io/nf-encycloped
 
 ## Development
 ### Running Tests
-We use the [pytest](https://docs.pytest.org/en/7.0.x/contents.html) Python package to run our tests. It can be installed either with either pip:
-
-```sh
-pip install pytest
-```
-
-or conda:
+We use the [pytest](https://docs.pytest.org/en/7.0.x/contents.html) Python package to run our tests. It can be installed using conda:
 
 ``` sh
+conda env create --name nf_encyclopedia --file=environment.yml
+conda activate nf_encyclopedia
 conda install pytest
 ```
 
-Once installed, tests can be run from the root directory of the workflow. These tests use the process stubs to test the workflow logic, but do not test the commands for the tools themselves. Run them with:
```

Comment:
It might be good to keep the information that the tests check the logic but not the tools.

Reply:
I removed it because right now some of the tests do run the tools:

- https://github.com/TalusBio/nf-encyclopedia/blob/main/tests/unit_tests/test_msstats.py
- https://github.com/TalusBio/nf-encyclopedia/blob/main/tests/system_tests/test_run.py

(sorry I should have mentioned that as well in the PR text, will edit it for the future)

Although ... it makes sure the flow works, but does not check the correctness of the input output ... LMK if I should add that and make clear that the processes are run and tested for succesful completion but not for correctness of output.